client/src/components/OpportunitiesList.js

```diff
@@ -92,7 +95,19 @@ class OpportunitiesList extends Component {
       formEntries: { ...this.state.formEntries, city: selectedCity }
     });
   };
+  filteringOpportunitiesBySkills = () => {

```

Comment:
rename this to `filterOppotunitiesBySkills` not `filtering` .. normally you just start with a very like `handle` etc..